### PR TITLE
chore(RHTAPWATCH-403): Update the Logs documentation

### PR DESCRIPTION
**Pull request by eisraeli**
Adding information about network logs.

### CODE REVIEW

File: book/logging.md [hunk 1]
```diff
@@ -11,85 +29,148 @@ regulatory requirements and uncovering the root causes of incidents.\
...
+As described in the [OpenShift Container Platform documentation](
+    https://docs.openshift.com/container-platform/4.13/logging/cluster-logging.html),
 the logging subsystem aggregates the following types of logs:
 
-_**application**_ - Container logs generated by user applications running in the cluster,
+**application** - Container logs generated by user applications running in the cluster,
 except infrastructure container applications.
 
-_**infrastructure**_ - Logs generated by infrastructure components running in the cluster
+*Note:* This includes controller audit logs are are mentioned on the
```
Comment (svatares) [inline book/logging.md:43]: Remove 1 of the `are`s in the sentence

File: book/logging.md [hunk 2]
```diff
@@ -1,4 +1,22 @@
-# RHTAP Logs
+# Logging
```
Comment (svatares) [inline book/logging.md:1]: Why removing the `RHTAP`?

File: book/logging.md [hunk 3]
```diff
@@ -11,85 +29,148 @@ regulatory requirements and uncovering the root causes of incidents.\
...
 
-* Secrets are also backed-up in the o11y team's BitWarden vault.
+**Notice** Secrets are also backed-up in the o11y team's BitWarden vault.
 
-### Splunk Related Links
-* [The Source - IT - Splunk](https://source.redhat.com/departments/it/splunk)
-* [Open Shift Documentation - Cluster Logging](https://docs.openshift.com/container-platform/4.13/logging/cluster-logging.html)
-* [Open Shift Cluster Logging Operator Github repository](https://github.com/openshift/cluster-logging-operator)
+### Logs retention
+Logs are being retained for 60 days.
```
Comment (svatares) [inline book/logging.md:170]: Is this configurable? and if so who and where it can be configured?

File: book/logging.md [hunk 4]
```diff
@@ -11,85 +29,148 @@ regulatory requirements and uncovering the root causes of incidents.\
...
+
+### Other links
+* [Applicative Log Conventions](https://github.com/redhat-appstudio/book/blob/main/ADR/0006-log-conventions.md#controller-audit-logs)
 
 ### Contact
 RHTAP logs are being managed by the o11y team.\
```
Comment (svatares) [inline book/logging.md:176]: What about adding a link to the o11y slack channel or mailing list?

**Review by svatares — COMMENTED** [review]
Looks good, left a few minor comments

File: book/logging.md [hunk 5]
```diff
@@ -11,85 +29,148 @@ regulatory requirements and uncovering the root causes of incidents.\
...
+For RHTAP, we use Splunk Cloud, which is a cloud-based version of the Splunk platform.
 It is being maintained by Splunk and Red Hat's IT Splunk team.
 
 RHTAP's Splunk: https://rhcorporate.splunkcloud.com
 
-### Logs retention in Splunk
+More information:
+* [The Source - IT - Splunk](https://source.redhat.com/departments/it/splunk)
+
+## Splunk - Operational Information
```
Comment (amisstea) [inline book/logging.md:120]: This section in particular seems far better suited to a series of SOPs. The title even suggests it with the term "Operational".

We already have an SOP for how to access the logs so why not put the rover group info and who to contact to gain access in there? The retention info could be a footnote on that SOP.

We could also have an SOP for rotating the secrets, if necessary. Documenting the vault locations here isn't very useful, IMO. Anyone can dig through the infra-deployments repo to figure these out but they generally shouldn't need to (as with all other vault secrets). It's the on-call person who may need to rotate a secret in a hurry who will need very explicit instructions.

**Review by amisstea — CHANGES_REQUESTED** [review]

File: book/logging.md (hunk 5 above)
Comment (arewm) [reply book/logging.md:120]: If we are going to remove this content to try to clean it to a more appropriate `book` audience, can we also take out Red Hat/RHTAP references?

Comment (amisstea) [reply book/logging.md:120]: Sorry, my bad. I'll save these comments for the followup PRs :smile:.

**Review by amisstea — COMMENTED** [review]
Just one small issue but otherwise LGTM.

File: book/logging.md [hunk 6]
```diff
@@ -11,85 +29,148 @@ regulatory requirements and uncovering the root causes of incidents.\
...
+performance, operational efficiency, application behavior, and other operational aspects. 
+
+### Splunk
 [Splunk](https://splunk.com) is a widely used log management and analytics platform that
 allows organizations to collect, index, search, analyze, and visualize machine-generated
 data, including logs from various sources. Splunk provides powerful search capabilities,
 real-time monitoring, alerting, and robust data visualization options.
 
-For RHTAP, we use a Splunk Cloud, which is a cloud-based version of the Splunk platform.
+For RHTAP, we use Splunk Cloud, which is a cloud-based version of the Splunk platform.
```
Comment (ifireball) [inline book/logging.md:112]: Are you planning to generalize this for AppStudio?

Comment (ifireball) [reply book/logging.md:117]: not sure about this, do you expect to merge this in a state that is not suitable for inclusion in the AppStudio book?

**Review by amisstea — APPROVED** [review]

File: book/logging.md (hunk 5 above)
Comment (arewm) [reply book/logging.md:125]: Great -- I wasn't aware of that conversation that happened.

**Review by ifireball — APPROVED** [review]

**Review by arewm — APPROVED** [review]

**Review by svatares — APPROVED** [review]

**Pull request merged**